specs/eip4844/fork-choice.md

```diff
@@ -23,54 +22,42 @@ This is the modification of the fork choice accompanying the EIP-4844 upgrade.
 
 ## Containers
 
-### `BlobsSidecar`
-
-```python
-class BlobsSidecar(Container):
-    beacon_block_root: Root
-    beacon_block_slot: Slot
-    blobs: List[Blob, MAX_BLOBS_PER_BLOCK]
-    kzg_aggregated_proof: KZGProof
-```
-
 ## Helpers
 
-#### `validate_blobs_sidecar`
+#### `validate_blob_sidecars`
 
 ```python
-def validate_blobs_sidecar(slot: Slot,
+def validate_blob_sidecars(slot: Slot,
                            beacon_block_root: Root,
                            expected_kzg_commitments: Sequence[KZGCommitment],
-                           blobs_sidecar: BlobsSidecar) -> None:
+                           blob_sidecars: Sequence[BlobSidecar]) -> None:
     assert slot == blobs_sidecar.beacon_block_slot
     assert beacon_block_root == blobs_sidecar.beacon_block_root
-    blobs = blobs_sidecar.blobs
-    kzg_aggregated_proof = blobs_sidecar.kzg_aggregated_proof
-    assert len(expected_kzg_commitments) == len(blobs)
-
-    assert verify_aggregate_kzg_proof(blobs, expected_kzg_commitments, kzg_aggregated_proof)
+    assert len(expected_kzg_commitments) == len(blob_sidecars)
```

Comment:
I suppose you want to verify a few more things here:
1. `blob_sidecars` is ordered by `BlobIndex`
2. the blobs are consistent with `expected_kzg_commitments`

Reply:
Since we're no longer passing sidecars, these checks are redundant / captured in the proof verification: da34af97d4b6575131e67c2bf22acb6de4f0951d